### PR TITLE
Chore/tri 944 documentation housekeeping

### PR DESCRIPTION
**Pull request by ds-jhartmann**

### CODE REVIEW

**Review by ds-jkreutzfeld — APPROVED** [review]

File: docs/src/docs/arc42/cross-cutting/safety-security.adoc [hunk 1]
```diff
@@ -15,9 +15,9 @@ The IRS acts as a client for the Digital Twin Registry (DTR), which is also secu
 In a decentralized network, IRS uses the EDC client to access the provider DTR. This way, no authentication, other than the EDC contract negotiation, is needed to access the DTR.
 
 === IRS as EDC client
-The IRS accesses the Catena-X network via the EDC consumer connector. This component requires authentication via a DAPS certificate, which was provided to the IRS via the network authority.
+The IRS accesses the Catena-X network via the EDC consumer connector. This component requires authentication via a Verifiable Credential (VC), which is provided to the EDC via the Managed Identity Wallet.
```
Comment (mkanal) [inline docs/src/docs/arc42/cross-cutting/safety-security.adoc:18]: Could you please add the new components to the glossar as well,

File: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc [hunk 2]
```diff
@@ -66,6 +66,7 @@ Data validation happens at two points:
 
 - IRS API: the data sent by the client is validated to match the model defined in the IRS. If the validation fails, the IRS sends a HTTP 400 response and indicates the problem to the caller.
 - Submodel payload: each time a submodel payload is requested from via EDC, the data is validated against the model defined in the SemanticHub for the matching aspect type.
+- EDC Contract Offer Policy: each time IRS consumes data over the EDC, the policies of the offered contract will be validated. Only policies which are defined via the PolicyStory will be accepted.
```
Comment (mkanal) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:69]: Add PolicyStore to Gloassar

Comment (mkanal) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:69]: Some more context on policies - We have Keys that are referencing "Rahmenverträge" which are checked in the eDC as Usagepolicies ...

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 3]
```diff
@@ -12,7 +12,7 @@ component [**IRSApplication**] <<System>> {
 
 component [**CatenaX-Network**] <<CX-Environment>> {
     component [**IAM/KeyCloak**] <<System>> as IAM_IRS
-    component [**IAM/DAPS**] <<System>> as IAM_DAPS
+    component [**MIW**] <<System>> as IAM_DAPS
```
Comment (mkanal) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:15]: IAM_DAPS should be renamed as the variable is pointing to a not existing concept . This migth be more generic like. SSI oder IDP

**Review by mkanal — COMMENTED** [review]
Some small changes

**Pull request merged**